itests/util/src/main/java/org/apache/hadoop/hive/cli/control/CliConfigs.java

```diff
@@ -240,6 +240,101 @@ public MiniLlapLocalCliConfig() {
     }
   }
 
+  public static class PostgresMetastoreCliConfig extends AbstractCliConfig {
+    public PostgresMetastoreCliConfig() {
+      super(CoreCliDriver.class);
+      try {
+        setQueryDir("ql/src/test/queries/clientpositive");
+        includesFrom(testConfigProps, "ms.postgres.query.files");
+        setResultsDir("ql/src/test/results/clientpositive/mspostgres");
+        setLogDir("itests/qtest/target/qfile-results/mspostgres");
+        setInitScript("q_test_init.sql");
+        setCleanupScript("q_test_cleanup.sql");
+        setHiveConfDir("data/conf/llap");
+        setClusterType(MiniClusterType.LLAP);
```

Comment:
Which is the least costly (wrt resources) cluster type?
Do we need to initialize LLAP for this?